pdns/dnsdist.hh

```diff
@@ -264,6 +265,18 @@ struct DownstreamState
   void setUp() { availability = Availability::Up; }
   void setDown() { availability = Availability::Down; }
   void setAuto() { availability = Availability::Auto; }
+  string toString() const {
```

Comment:
ok, this confused me - I think it would be better to just make this getName() and that this is then documented to return the toStringWithPort() if you haven't set a name yourself. It is a bit strange to me to to toStringWithPort() actually return a name, and not an IP address. Sorry for picking this nit!